app/cdap/components/WranglerGrid/AddTransformationPanel/SelectColumnsWidget/index.tsx

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React from 'react';
+import T from 'i18n-react';
+import {
+  MULTI_SELECTION_COLUMN,
+  ADD_TRANSFORMATION_PREFIX,
+} from 'components/WranglerGrid/SelectColumnPanel/constants';
+import { SubHeadBoldFont, NormalFont } from 'components/common/TypographyText';
+import { SelectColumnButton } from 'components/common/ButtonWidget';
+import styled from 'styled-components';
+import { IHeaderNamesList } from 'components/WranglerGrid/SelectColumnPanel/types';
+
+interface ISelectColumnsWidgetProps {
+  selectedColumns: IHeaderNamesList[];
+  transformationName: string;
+  handleSelectColumn: (value: boolean) => void;
+}
+
+export const TickIcon = (
```

Comment:
move this svg to icons similar to underline icon
plz check with siva if you need help, he has done it before

plz move all svgs from this feature code to icons